pnpm-lock.yaml

```diff
@@ -207,7 +207,7 @@ importers:
         version: 5.0.4
       zod:
         specifier: ^3.20.2
-        version: 3.20.2
```

Comment:
checkout the lockfile to main to revert it completely. now we actually bumped it anyways.